Automattic/wp-calypso
# Stats: Add new SubscribersHighlightSection component to Subscribers page

Related to #75855.

## Proposed Changes

Replaces the `AnnualHighlightsSection` component (from the Traffic page) with a new `SubscribersHighlightSection` component specific to this page.

- Mostly lines up with the designs in 75855 except for the number formatting. Currently using the same formatting rules as the Traffic page for consistency.
- Not wired to any real data yet. Just using hard-coded data per the initial designs.

Will address the following (as needed) in follow-up PRs:

1. Tooltip content for "i" icon.
2. Do we want tooltips on the individual values? Currently enabled.
3. Do we want icons for each card?

## Testing Instructions



* Open the Live branch.
* Visit Jetpack → Stats.
* Append `?flags=stats/subscribers-section` to the URL and load.
* Visit the new Subscribers page.
* Confirm the new "All time stats" section is visible and behaves as expected.

<img width="753" alt="SCR-20230419-qvga" src="https://user-images.githubusercontent.com/40267301/233060017-038e5185-9e4a-431a-be2d-8fc936fc89a2.png">

## Pre-merge Checklist
